### DISCUSSION TITLE
Resize Live rtsp stream and re-serve using ffmpeg and rtsp-simple-server

### DISCUSSION THREAD

**Discussion by JamesMcGloin** [bluenviron/mediamtx#709] [Questions]
Hi All, I am a newbie, I am trying to use rtsp-simple-server to re-transmit a re-sized stream using ffmpeg, I do realize that the resizing would be on a frame by frame bases after decoding and re-encoding and a new  rtsp stream would need to be generated.

This works: (rtsp pass through)
ffmpeg -y -i rtsp://<my-input-rtsp stream> -f rtsp://localhost:8554/mystream

This does not work: (rtsp resizing)

ffmpeg -i 'rtsp://<my-input-rtsp stream>' -s 480x272 -crf 30 -preset ultrafast -vcodec libx264 -r 25 -b:v 500k -f rtsp rtsp://localhost:8554/mystream

error: [rtsp @ 0x938d10] Estimating duration from bitrate, this may be inaccurate

I am sure I am missing something.

Would appreciate any help on this.

James

**Comment by aler9** [comment]
Hello, the command you posted works perfectly.

[a comment by an automated account is not shown]
